docs/config/configuration.md

```diff
@@ -143,6 +143,11 @@ bitcoin-s {
 
     # settings for wallet module
     wallet {
+        # You can have multiple wallets by setting a different
+        # wallet name for each of them. They will each have
+        # their own seed and database.
```

Comment:
nit: In the case of sqlite each wallet will have it's own database, in the case of postgres backend each wallet will have it's own schema. Both will have seeds placed in `datadir/seeds/`

Reply:
Also are there any known restrictions on naming a wallet ? Can i use `-` `?` `_` etc